cmd/init.go

```diff
@@ -43,6 +44,112 @@ func (h *Handler) initNew(ctx context.Context, req *entity.CommandRequest) error
 	return nil
 }
 
+func (h *Handler) initFromTemplate(ctx context.Context, req *entity.CommandRequest) error {
+	// Prompt for a template to use
+
+	template, err := ui.PromptStarterTemplates()
+	if err != nil {
+		return err
+	}
+
+	// Select GitHub org
+
+	ui.StartSpinner(&ui.SpinnerCfg{Message: "Fetching GitHub scopes"})
+	scopes, err := h.ctrl.GetWritableGithubScopes(ctx)
+	if err != nil {
+		return err
+	}
+	if len(scopes) == 0 {
+		return errors2.NoGitHubScopesFound
+	}
+	ui.StopSpinner("")
+
+	org, err := ui.PromptGitHubScopes(scopes)
+	if err != nil {
+		return err
+	}
+
+	// Enter project name
+
+	name, err := ui.PromptProjectName()
+
+	// Prompt for env vars (if required)
+
+	variables := make(map[string]string)
+	if len(template.EnvVars) > 0 {
+		fmt.Printf("\n%s\n", ui.Bold("Environment Variables"))
+		for _, envVar := range template.EnvVars {
+			fmt.Printf("\n(%s)\n", ui.GrayText(envVar.Desc))
+			v, err := ui.PromptText(envVar.Name)
+			if err != nil {
+				return err
+			}
+			variables[name] = v
+		}
+		fmt.Print("\n")
+	}
+
+	// Create Railway project
+
+	ui.StartSpinner(&ui.SpinnerCfg{Message: "Creating project"})
```

Comment:
Same linesplit nit. Your call obvs
```suggestion
	ui.StartSpinner(&ui.SpinnerCfg{
	    Message: "Creating project",
	})
```